c/meson.build

```diff
@@ -90,9 +89,14 @@ configure_file(input: 'capi-ml-inference.pc.in', output: 'capi-ml-inference.pc',
   configuration: api_conf
 )
 
+ml_api_conf = configuration_data()
```

Comment:
recommend to move this and change .pc also (line 89 configuration: api_conf > ml_api_conf)

Reply:
Thank you for the suggestion! updated as per offline discussion

Reply:
Thank you for the suggestion! Changed as per offline discussion :)